tools/dockerfile/grpc_artifact_centos6_x64/Dockerfile

```diff
@@ -0,0 +1,58 @@
+# Copyright 2016 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Docker file for building gRPC artifacts.
+
+##################
+# Base
+
+FROM dockcross/manylinux2010-x64
+
+# Install essential packages.
+RUN yum -y install golang strace
+
+
+##################
+# Ruby dependencies
+
+# Install rvm
+RUN gpg --keyserver hkp://pool.sks-keyservers.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
```

Comment:
why do you need ruby and PHP if this is only used for building C# artifacts?

Reply:
This is for both .NET and PHP. Ruby might be removed from here but I just copied everything from `grpc_artifact_linux_x64/Dockerfile`.